messages/progress-event-listener.md

```diff
@@ -13,6 +16,5 @@ Executing rules
 # execution-spinner.engine-status
  - %s at %d% completion.
 
-# base-spinner.done
-done
-
+# execution-spinner.finished-status
+done. Executed rules from %s.
```

Comment:
Since the spinner already uses "Executing rules", it seemed correct to use "Executed rules" here instead of something like "Ran".